command/agent/template/template.go

```diff
@@ -0,0 +1,186 @@
+// Package template is responsible for rendering user supplied templates to
+// disk. The Server type managing the lifecycle of an internal Consul Template
+// Runner
```

Comment:
Seems like a sentence fragment.

Reply:
Yep! I reworded this in 14daed773 , please let me know what you think